sdk/maps/azure-maps-search/README.md

```diff
@@ -26,47 +26,27 @@ pip install --pre azure-maps-search
 
 ### Authenticate the client
 
-In order to interact with the Document Translation feature service, you will need to create an instance of a client.
-An **endpoint** and **credential** are necessary to instantiate the client object.
+In order to interact with the Azure Maps Search service, you will need to create an instance of a client.
+An **client_id** and **credential** are necessary to instantiate the client object.
 
 ### Create the client with AzureKeyCredential
 
-To use an [subscription_key] as the `credential` parameter,
+Once completed, set the values of the client ID, and subscription key as environment variables:
+`AZURE_CLIENT_ID`, `AZURE_SUBSCRIPTION_KEY`.
+
+To use an `AZURE_SUBSCRIPTION_KEY` as the `credential` parameter,
 pass the key as a string into an instance of [AzureKeyCredential][azure-key-credential].
 
 ```python
 from azure.core.credentials import AzureKeyCredential
 from azure.maps.search import SearchClient
 
-endpoint = "https://<resource-name>.mapsservices.azure.com/"
-credential = AzureKeyCredential("<subscription_key>")
-search_client = SearchClient(endpoint, credential)
-```
-
-### Create the client with an Azure Active Directory credential
```

Comment:
Why this section is deleted? We should let use able to use AAD to send requests, and this is a more recommendable way for authentication.